scripts/aws/configure

```diff
@@ -1,40 +1,43 @@
-#!/bin/sh
```

Comment:
Did you write all of these scripts in the `scripts` directory from scratch? V impressive.